compleet-client/src/ui/menu.rs

```diff
@@ -174,8 +175,26 @@ pub fn find_position(
 /// Returns the number of screen rows above and below the current cursor
 /// position.
 fn rows_above_below_cursor(lua: &Lua) -> LuaResult<(u16, u16)> {
-    let total_rows = api::get_option::<u16>(lua, "lines")?;
-    let rows_above = r#fn::screenrow(lua)? - 1;
+    let lines = api::get_option::<u16>(lua, "lines")?;
+    let cmdheight = api::get_option::<u16>(lua, "cmdheight")?;
+    let laststatus = api::get_option::<u16>(lua, "laststatus")?;
+    let tabline = api::get_option::<String>(lua, "tabline")?;
```

Comment:
Check `showtabline` instead of `tabline` to see if the tabline takes up a row

Reply:
What would you recommend for integer types?

Reply:
`showtabline` is either 0, 1 or 2 so a `u8` is more than enough.

Reply:
Also I was thinking `u16` should be enough for `row`and `col` right?

Reply:
No, `row` and `col` can be negative

Reply:
my bad, I meant `rowabove`, `colbefore` etc...

Reply:
Yes, aren't those `u16` already?

Reply:
Yup, just wanted to make sure.